src/2XXXR0_a_motivation_scope_and_plan_for_a_physical_quantities_and_units_library.md

```diff
@@ -44,8 +44,11 @@ the best we can get today in the C++ language.
 
 ## Dominik Berner
 
-Author of [@SI_LIB].
+Dominik is a strong believer that the C++ language can provide very high safety guarantees when programming through strong typing - A type error catched during compilation saves hours of debugging. Since the last 15 years he mainly codes in C++ an actively follows its evolution through the new standards. 
 
+When working in Med-Tech on regulated projects there usually where very close requirements on which datatypes were to be used for what, however these eventually were just lists of primitives to be memorized by each developer. However throughout his career Dominik spent way too many hours debugging and fixing issues caused by these types being incorrect. In an attempt to bring a closer semantic meaning to these list he eventually wrote [@SI_LIB] as a side project.
+
+While SI provides many useful features such as type safe conversion between physical quantities as well as zero-overhead computation for values of the same units, there are some shortcomings which would require major rework. Instead of creating yet another library Dominik decided to join forces with the other authors of this paper to push for standardizing support for more type-safety for physical quantities. He hopes that this will eventually lead to a safer and more robust C++ and open many more opportunities for the language.
```

Comment:
```suggestion
While SI provides many useful features such as type safe conversion between physical quantities as well as zero-overhead computation for values of the same units, there are some shortcomings which would require major rework. Instead of creating yet another library, Dominik decided to join forces with the other authors of this paper to push for standardizing support for more type-safety for physical quantities. He hopes that this will eventually lead to a safer and more robust C++ and open many more opportunities for the language.
```